build.gradle

```diff
@@ -111,6 +112,12 @@ tasks.check.dependsOn integrationTest
 
 gradlePlugin {
     testSourceSets project.sourceSets.integration
+		plugins {
```

Comment:
It was just to get it to build locally. I'll update this and see where it goes from there.